backend/fms_core/viewsets/_constants.py

```diff
@@ -302,6 +302,8 @@
     "sample__derived_samples__biosample__individual__name": CATEGORICAL_FILTERS_LOOSE,
     "sample__container__name": CATEGORICAL_FILTERS_LOOSE,
     "sample__container__barcode": CATEGORICAL_FILTERS_LOOSE,
+    "sample__container__location__barcode": CATEGORICAL_FILTERS_LOOSE,
+    **_prefix_keys("sample__container__location__coordinate__", _coordinate_filterset_fields),
```

Comment:
coordinate is on container not on location this would get you the parent parent coordinate